### ISSUE TITLE
Charts 11.11 - further Customization

### ISSUE THREAD

**Issue by todayifeeltired** [HanSolo/charts#76]
Hi there (again) :D
Thanks again for the quick implementation of issue #74 . I am very happy with it :)

I have one more wish, if that's not too much to ask for.
I´d like to have another StringProperty on the chartitem (and another constructor).
Heres why:
i got some data that i want to show on the coxcombchart that has a very long name.
i´d like to show a "succintly name" on the chart and on a label (where obviously would be enough space of showing it), the "long name".

Basically all i´d need is a stringproperty on the chartitem, a constructor so i can fill it and a getter for the new StringProperty.
The constructor should look something like this:
`public ChartItem(String NAME, double VALUE, String LONGNAME, Color FILL)`
"LONGNAME" might not be the best choice of name, but i think you get what i mean :D

this seems more like an feature request (for which I did not find the possibility).

**Comment by todayifeeltired** [comment]
oh btw:
In case you just forgot to answer it: i´d like paypal a beer/coffee to you for the work i was/am causing

**Comment by HanSolo** [comment]
Hi there, well no worries, all good. Currently I‘m on travel in the US so I won‘t have a lot pf time to look at that other request but it’s not forgotten 😁

**Comment by todayifeeltired** [comment]
then just enjoy the vacation for now, have a good one :)

**Comment by HanSolo** [comment]
Sorry for the late reply, I'm currently in Dubai but finally found some time to work on your request. In the latest commit I've added a Metadata interface. In the ChartItem class you now will find a property for Metadata with a getter, setter and a property method. There is also one constructor which supports it but also the ChartItemBuilder has a metadata method. I've added an example to the CoxcombChartTest to give you an idea on how to use it. Please let me know if this solves your problem. With this approach one can add whatever class you want as long as it implements the Metadata interface.

**Comment by todayifeeltired** [comment]
No worries, thanks for getting back to me :)

This solves my problem very well, love the approach!
Would appreciate it, if you release it on jdk11.

out of interest: have you been to the usa and dubai for vacation or business?

**Comment by HanSolo** [comment]
Ok…will add it to the jdk11 branch later today. Trips are all business 😁

**Comment by todayifeeltired** [comment]
thanks for your work and willigness to enhance your project. 
i really love it and appreciate it.

everything works fine :)!

**Comment by HanSolo** [comment]
👍🏻😁
